middleware/redis_manager.py

```diff
@@ -0,0 +1,47 @@
+from datetime import datetime
+from django.conf import settings
+from django.core.cache import cache
+from django_redis import get_redis_connection
+from middleware.utils import get_current_truncated_utc_z
+
+
+class RedisManager:
```

Comment:
RedisManager doesnt align with what we are using the class for, try a different name, maybe StatusLogManager?

Reply:
recently we starting using this manager now for storing observations too not just camera and device statuses should we still rename it ?